x-pack/plugins/ml/public/components/field_title_bar/field_title_bar.js

```diff
@@ -33,7 +33,7 @@ export function FieldTitleBar({ card }) {
 
   return (
     <EuiText className={classNames.join(' ')}>
-      <FieldTypeIcon type={card.type} tooltipEnabled={true} />
+      <FieldTypeIcon type={card.type} tooltipEnabled={true} ariaSuffix={fieldName} />
```

Comment:
I think a cleaner way of doing this, rather than passing the name of the field to an icon whose purpose is only to indicate the 'type' of the field, would be to make the `fieldName div` keyboard accessible. Maybe this could be done by wrapping the `field-name div` in an `EuiKeyboardAccessible` element?

Reply:
@peteharverson, for updating you on your suggestion:
Unfortunately, it's impossible to use `EuiKeyboardAccessible` for this purpose since
```
Error: Warning: Failed prop type:
EuiKeyboardAccessible needs to wrap an element which has an onClick prop assigned.
```
however, we don't need to use onClick() functionality.
So, I'm using just a tabIndex property.